hugegraph-llm/src/hugegraph_llm/demo/rag_web_demo.py

```diff
@@ -141,13 +134,110 @@ def build_kg(file, schema, example_prompt, build_mode):  # pylint: disable=too-m
         raise gr.Error(str(e))
```

Comment:
fixme: return 'str',  but `context` got 'dict[str, Any]'... (unify it)

<img width="691" alt="image" src="https://github.com/user-attachments/assets/4e20639c-7548-4d5e-8234-2c7888c5a131">